numpy/polynomial/_polybase.py

```diff
@@ -685,9 +685,11 @@ def degree(self):
 
         Create a polynomial object for ``1 + 7*x + 4*x**2``:
 
+        # set the print format to ascii for doctests
+        >>> np.polynomial.set_default_printstyle("ascii")
```

Comment:
I could have used 'unicode' but I am not sure what windows CI does...

Reply:
For this one, I think it would be nice to keep the unicode, because users are more likely to see it then not, I believe.

I suspect `numpy/conftest.py` is read?  Maybe we can do `p.polynomial.set_default_printstyle("unicode")` there to remove windows from the equation?
(I think different defaults is just about bad terminals, which might print oddly if the test fails, but shouldn't influence test results.)

But if this isn't trivial, we can also just do this.

Reply:
I changed it to unicode and checked that it works on my windows 10 machine. I prefer to leave the `np.polynomial.set_default_printstyle` in the example so it is more obvious there is an alternative. MAybe @rossbar knows why windows defaults to `ascii`?

Reply:
OK, maybe @rossbar can follow up if he prefers a different approach, would be nice to get this in.

IIRC, some windows terminals (older powershell?) have poor unicode printing support (it may even have just been the font missing symbols).

The overview docs and the docs of `set_default_printstyle` itself mention it, so I think that setting it globally would be OK, but then those two places should make sure to reset it to whatever it (with a comment that docs use unicode).